indico/modules/designer/controllers.py

```diff
@@ -324,10 +325,11 @@ def _process(self):
             return jsonify(error='File format not accepted!')
         content_type = 'image/' + image_type
         image = DesignerImageFile(template=self.template, filename=filename, content_type=content_type)
-        self.template.background_image = image
+        if request.form.get('for_background') == 'True':
```

Comment:
Instead of checking for an input value `for_background`, I would recommend changing a bit the logic here to one of the following:
- We could rename this endpoint to `RHUploadImage` and have another dedicated one for setting the background image `RHSetBackgroundImage`
- Alternatively, we could add a query param `background=True/False` when we intend to set a background. The form action would be: 
```
action="{{ url_for('designer.upload_image', template, background=True) }}"
```

Reply:
Okay, thank you